docs/architecture/event-flows-proposal.md

```diff
@@ -0,0 +1,85 @@
+# Eventing flow proposal
+
+## Overview
+
+This document shows a proposal of two ways of eventing flow we want to support with use of the [Management Plane](./terminology.md#management-plane)
+
+## Eventing with Kyma internal Event Bus
+
+![Management Plane Components](assets/internal-event-flow.svg)
```

Comment:
Diagram, "register event with type of internal communication" (that means, http, mqtt, auth etc). Make same descriptions in both diagrams if they are the same.

Reply:
changed it